lib/src/gainmapmath.cpp

```diff
@@ -66,28 +66,37 @@ void ShepardsIDW::fillShepardsIDW(float* weights, int incR, int incB) {
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// To compute luminance for a set of primaries and RGB Tuple,
+// See https://registry.khronos.org/DataFormat/specs/1.3/dataformat.1.3.html#PRIMARY_CONVERSION
+
+// For RGB <-> YCbCr conversions for a set of primaries,
+// See https://registry.khronos.org/DataFormat/specs/1.3/dataformat.1.3.html#MODEL_YUV
+
 ////////////////////////////////////////////////////////////////////////////////
 // sRGB transformations
 
-// See IEC 61966-2-1/Amd 1:2003, Equation F.7.
-static const float kSrgbR = 0.2126f, kSrgbG = 0.7152f, kSrgbB = 0.0722f;
+// BT709 Luminance
+// See https://registry.khronos.org/DataFormat/specs/1.3/dataformat.1.3.html#PRIMARIES_BT709
+// BT709 chromaticity coordinates are also shared by sRGB
```

Comment:
I would recommend keeping the original source. It's OK to keep both sources, so that this is a mark that we confirmed the numbers with two sources.

Reply:
And same for the rest

Reply:
Yes Dichen, made suggested changes.